pkg/diff/sql_generator.go

```diff
@@ -14,6 +14,8 @@ const (
 	maxPostgresIdentifierSize = 63
 
 	statementTimeoutDefault = 3 * time.Second
+	lockTimeoutDefault      = statementTimeoutDefault
```

Comment:
Basically modified with the regex above:
`s/Timeout: (.*)/Timeout: $1,\nLockTimeout: lockTimeoutDefault/g`

It might be worth switching the statement creation over to a constructor:
`newStatement(ddl string, opts statementOpt)`, which sets the default statement and lock timeouts.  I'm a little hesitant to do this because it creates more indirection